openscm/parameter_views.py

```diff
@@ -111,6 +112,29 @@ def get_series(self) -> Sequence[float]:
         """
         Get values of the full timeseries.
         """
+        if self.is_empty:
+            if not self._parameter._children:
+                raise ParameterEmptyError
+
+            def get_child_data(para):
+                # where should this go?
```

Comment:
@swillner parameters.py line 207?

Reply:
yes, that should be a method of the `Parameter` class